packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -292,6 +319,83 @@ const Autocomplete = <
       testId,
     ]
   );
+
+  const renderVirtualizedRow = ({
+    data,
+    index,
+    style,
+  }: ListChildComponentProps) => {
+    const baseOption = data[index];
+    /**
+     * react-window calculates the absolute positions of the list items, via an inline style, so
+     * we need to add it to each list item that is being rendered in the viewable list window.
+     * See here if you need to know more: https://github.com/bvaughn/react-window?tab=readme-ov-file#why-is-my-list-blank-when-i-scroll
+     */
+    const optionItem = { ...baseOption, props: { ...baseOption.props, style } };
+    return optionItem;
+  };
+
+  const OuterListboxContext = createContext({});
+
+  const OuterListboxElementType = React.forwardRef<HTMLDivElement>(
+    (props, ref) => {
+      const outerProps = React.useContext(OuterListboxContext);
+      return <div ref={ref} {...props} {...outerProps} />;
+    }
+  );
+
+  function useResetCache(length: number) {
+    const ref = React.useRef<VariableSizeList>(null);
+    useEffect(() => {
+      if (ref.current != null) {
+        ref.current.resetAfterIndex(0, true);
+      }
+    }, [length]);
+    return ref;
+  }
+
+  const ListboxComponent = forwardRef<
+    HTMLDivElement,
+    React.HTMLAttributes<HTMLElement>
+  >(function (props, ref) {
+    const { children, ...other } = props;
+    const itemData: React.ReactElement[] = [];
+    (children as React.ReactElement[]).forEach(
+      (item: React.ReactElement & { children?: React.ReactElement[] }) => {
+        itemData.push(item);
+        itemData.push(...(item.children || []));
+      }
+    );
+
+    const optionHeight = 45; //px
+    const gridRef = useResetCache(itemData.length);
+
+    return (
+      <div ref={ref} style={{ width: "100%", height: "100%" }}>
+        <OuterListboxContext.Provider value={other}>
+          <AutoSizer>
+            {({ height, width }) => (
+              <VariableSizeList
+                innerElementType="ul"
+                itemData={itemData}
+                itemCount={itemData.length}
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                itemSize={(_index) => optionHeight}
```

Comment:
Does it let us remove the unused arg, or does it require it to be there?
```suggestion
                itemSize={() => optionHeight}
```